pyproject.toml

```
@@ -157,6 +157,7 @@ dependencies = [
   "siphashc>=2.1,<3.0",
   "social-auth-app-django>=5.4.2,<6.0.0",
   "social-auth-core>=4.5.4,<5.0.0",
+  "standardwebhooks>=1.0.0",
```

Let's add upper bound so that we can notice API breakages.

```suggestion
  "standardwebhooks>=1.0.0,<1.1",
```